### DISCUSSION TITLE
Add --loader to package.json

### DISCUSSION THREAD

**Discussion by Setitch** [nodejs/help#4667] [Ideas]
Idea of `--loader`s is neat. But i think it would be great addition if some default loaders could be configured on `package.json` level

**Comment by AugustinMauroy** [comment]
Super interesting, but could you give an example of `package.json`.

**Reply by Setitch** [reply]
For now i was thinking about something like that
```json5
// ...,
  loaders: [
    "pnp", {"loader": "ts-node", "args": "additional arguments? maybe also in array"}
 ],
// ....
  ```
Basically: array of loaders that can be given as string (name of loader)/package
But also (dont know if that would be needed but its good to think of it already), advanced configuration with object providing the string for name/package but also additional arguments or other parameters if required
